grid/chat/delete-message/maps/delete-message.ts

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+export const deleteMessageTest = (
```

Comment:
I saw this in 283 already :|

Reply:
It's weird that it does not recognise that it was created in 283 😕